controllers/operandconfig.go

```diff
@@ -141,8 +141,18 @@ func mergeChangedMap(key string, defaultMap interface{}, changedMap interface{},
 			//Check if the value was set, otherwise set it
 			if changedMap == nil {
 				finalMap[key] = defaultMap
+			} else if defaultMap == nil {
+				// need to handle case where user explicitly set value to null
+				finalMap[key] = changedMap
```

Comment:
@bitscuit Could you explain why this is required?

From my understanding, finalMap[key] is always equal to changedMap.

Reply:
good catch. Will remove it. It was there initially to solve an issue, but the comparable key check solves it as well.